core/src/main/java/io/dekorate/utils/Images.java

```diff
@@ -103,12 +103,24 @@ public static String getName(String image) {
   /**
    * Return the tag of the image.
    * @param image The docker image.
-   * @return The tag if present or null otherwise.
+   * @return The tag if present, {@code latest} otherwise.
    */
   public static String getTag(String image) {
     if (image.contains(COLN)) {
       return image.substring(image.indexOf(COLN) + 1);
     }
+    return "latest";
```

Comment:
Alternatively, we could make it return `null` as the javadoc used to say, but then all consumers (of which there's 1, AFAIK) would have to handle that. I think `latest` is the customary default value.